optics/core/src/main/scala/tofu/optics/functions.scala

```diff
@@ -56,4 +56,6 @@ object functions {
   def some[A]: Subset[Option[A], A] = Subset[Option[A]](identity)(Some(_))
 
   def none[A]: Subset[Option[A], Unit] = Subset[Option[A]](_ => Some(()))(_ => None)
+
+  def extractAnyFromNothing: Extract[Nothing, Any] = (s: Nothing) => s
```

Comment:
Well, `Extract[A, B]` unlike `Lens[A, B]` is perfectly legal  for any `A <: B`, so it would be better to see more general case here.